docs/omero_figure_scripting.rst

```diff
@@ -63,16 +59,32 @@ We will use the time-lapse images listed above to create a FRAP figure but you c
         "x": 200, "y", 200, "width": 100, "height": 100,
         ...many other attributes not shown...
 
-#.  The ``figureModel`` variable is accessible in the console. We can use ``figureModel.getSelected()`` to get selected panels and for each panel we can call ``p.set()`` to change an attribute.
+#.  The ``figureModel`` variable is accessible in the `Console` of the browser `Developer Tools`. The easiest way to open
+    the developer tools in most browsers is to open the context menu (right-click) anywhere on the page and choose `Inspect`.
+
+#.  We can use ``figureModel.getSelected()`` to get selected panels and for each panel we can call ``p.set()`` to change an attribute.
 
-#.  For example, to set the ``height`` of each selected panel to ``200``, we can do:
+#.  For example, to set the ``height`` of several panels to ``200``, we can select the panels in the figure UI and paste
+    this code snippet in the `Console`:
```

Comment:
For the code snippet below, when the workflow was followed as written, I get just

```
Location: https://workshop.openmicroscopy.org/figure/load_web_figure/136949/?_=1622452009145
Line Number 1, Column 1: 136949:1:1
Empty string passed to getElementById().
```

Reply:
This code snippet works in Chrome. But in my FF, I am getting the problem described in https://github.com/ome/omero-guide-figure/pull/11/files#r642426364 - I guess this functionality is Chrome only or some steps are missing in description.